lib/galaxy/tools/parameters/basic.py

```diff
@@ -473,6 +475,12 @@ def from_html( self, value, trans=None, other_values={} ):
     def to_python( self, value, app=None ):
         return ( value in [ True, 'True', 'true' ] )
 
+    def to_json( self, value, app=None ):
+        if value is True:
+            return 'true'
+        else:
+            return 'false'
```

Comment:
Yes, we can just return the boolean which will then be implicitly converted into the corresponding string at: https://github.com/galaxyproject/galaxy/blob/dev/client/galaxy/scripts/mvc/ui/ui-options.js#L281